src/Nether.Analytics.DataLake/Nether.Analytics.DataLake.csproj

```diff
@@ -19,10 +19,14 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.DataLake.Analytics" Version="2.2.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.DataLake.Store" Version="2.1.3-preview" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="1.1.2" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="1.1.2" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="1.1.2" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.6" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.Authentication" Version="2.2.12" />
     <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
     <PackageReference Include="System.Net.Http" Version="4.3.2" />
+    <PackageReference Include="WindowsAzure.Storage" Version="8.1.1" />
```

Comment:
If possible move all dependencies to AzureStorage out of this project into the project designated to interaction with storage.